src/client/components/EventDepartments.js

```diff
@@ -59,24 +65,76 @@ const DepartmentList = ({ departments }) => (
 );
 
 const SummaryRow = ({ eventId }) => (
-  <div className={grid.row}>
-    <section className={grid.col_sm_4} style={{position: 'relative'}}>
-      <PageTitle className={theme.txt_lightest}
-                 title={<FormattedMessage id="event.teams" defaultMessage="Departments & Teams" />}
-                 link={`/event/${eventId}/overview`} />
-      <DataLoader serviceCall={`events/${eventId}/lead`} component={LeadInfo} />
-    </section>
-    <section className={grid.col_sm_8}>
-      <div className={concat(style.info, theme.bg_content)}>
-        <DepartmentNumbers />
-        <DataLoader serviceCall={`events/${eventId}/departments`} component={DepartmentList} />
-      </div>
-    </section>
+  <div className={theme.page_padding}>
+    <div className={grid.row}>
+      <section className={grid.col_sm_4} style={{position: 'relative'}}>
+        <PageTitle className={theme.txt_lightest}
+                   title={<FormattedMessage id="event.teams" defaultMessage="Departments & Teams" />}
+                   link={`/event/${eventId}/overview`} />
+        <DataLoader serviceCall={`events/${eventId}/lead`} component={LeadInfo} />
+        <Button text={<FormattedMessage
+                          id="department.create"
+                          defaultMessage="Create New Department"/>}
+                icon={addSvg}
+                className={concat(style.button, theme.bg_3, theme.txt_lightest)}
+        />
+        <Button text={<FormattedMessage
+                          id="department.edit"
+                          defaultMessage="Edit Departments"/>}
+                icon={editSvg}
+                className={concat(style.button, theme.bg_2, theme.txt_lightest)}
+        />
+      </section>
+      <section className={grid.col_sm_8}>
+        <div className={concat(style.info, theme.bg_content)}>
+          <DepartmentNumbers />
+          <DataLoader serviceCall={`events/${eventId}/departments`} component={DepartmentList} />
+        </div>
+      </section>
+    </div>
+  </div>
+);
+
+const Departments = ({ }) => (
+  <div className={theme.page_padding}>
+    <div className={theme.bg_content} style={{ height: '100px' }}>
+    </div>
   </div>
 );
 
+class SearchForm extends React.Component {
+  constructor() {
+    super();
+    this.state = { search: '' };
+  }
+
+  updateSearch(value) {
```

Comment:
So the JS convention for "setter" methods is "updateX", not "setX"?
_Please_ say it ain't so :(

Reply:
I don't know about JS convention, but I do tend to use 'update' over 'set'. I never noticed that before.

Reply:
Use 'set' then please - it's the convention for setters in every single self-respecting object-oriented language out there.